lib/src/main/java/com/basistheory/android/model/ElementValueReference.kt

```diff
@@ -1,13 +1,20 @@
 package com.basistheory.android.model
 
+import com.basistheory.android.constants.ElementValueType
 import com.basistheory.android.model.exceptions.IncompleteElementException
 import com.basistheory.android.view.TextElement
 
 class ElementValueReference(private val valueGetter: () -> String?) {
     private var _element: TextElement? = null
+    private var _getValueType: ElementValueType? = ElementValueType.STRING;
 
-    constructor(element: TextElement, valueGetter: () -> String?) : this(valueGetter) {
+    constructor(
+        element: TextElement,
+        valueGetter: () -> String?,
+        getValueType: ElementValueType?
+    ) : this(valueGetter) {
         _element = element
+        _getValueType = getValueType
```

Comment:
Should this fallback to STRING if null? It also might be more kotlin-like to make the property `getValueType` non-nullable here, and if you get null provided in the constructor, then fallback to ElementValueType.STRING like we have as the default, instead of setting the property to null.